config/config.go

```diff
@@ -31,6 +34,12 @@ type Generator struct {
 	Out       string
 }
 
+type Convertor struct {
+	Params     []KeyValue
+	ImportPath string
```

Comment:
I was actually imagining this to be a list. Imagine that we want to find proto files in multiple places, like:

```
import_path=our/proto/files
import_path=vendored/proto/files
```

This doesn't have to work today, but I think limiting it to zero or one directories is a bit weird.

Reply:
It seems that currently the dep used to parse `ini` files doesn't provide a way to have the same key multiple times. It will always return `our/proto/files`

Reply:
We've agreed to look into this in a future PR.